File: trunk/Robustez/Robustez/Arista.cs
namespace Robustez
{
    public class Arista<T>
    {

        private Vertice<T> _origen;
        private Vertice<T> _destino;

        public Vertice<T> Origen
        {
            get { return _origen; }
            set { _origen = value; }
        }

        public Vertice<T> Destino
        {
            get { return _destino; }
            set { _destino = value; }
        }


        public Arista(Vertice<T> verticeCicloUno, Vertice<T> verticeCicloDos)
        {
            Origen = verticeCicloUno;
            Destino = verticeCicloDos;
        }


    }
}

File: TeoriaDeAlgoritmos/src/main/java/ar/fi/uba/tda/colecciones/Arista.java
package ar.fi.uba.tda.colecciones;

public class Arista {

	private Vertice<?> origen;
	private Vertice<?> destino;
	
	
	public Arista(Vertice<?> verticeCicloUno, Vertice<?> verticeCicloDos) {
		this.origen = verticeCicloUno;
		this.destino = verticeCicloDos;
	}

	public Vertice<?> getOrigen() {
		return origen;
	}

	public Vertice<?> getDestino() {
		return destino;
	}

}
